setup.cfg

```diff
@@ -75,6 +75,7 @@ tests =
     hera_cal @ git+git://github.com/hera-team/hera_cal
     healvis @ git+git://github.com/rasg-affiliates/healvis
     bda @ git+git://github.com/HERA-Team/baseline_dependent_averaging.git
+    vis_cpu >= 0.2.0
```

Comment:
Conflict between vis_cpu version listed here and the version listed in the `install_requires` section.